product_docs/docs/biganimal/release/getting_started/identity_provider/index.mdx

```diff
@@ -34,36 +34,42 @@ http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress
-   - `surname` &mdash; BigAnimal uses the value as the surname in the profile of the authenticated user.
-   - `name` &mdash; BigAnimal uses the value as the full name (`givenname` joined with `surname`) in the profile of the authenticated user.
-   - `emailaddress` (optional) &mdash; BigAnimal uses the value for the email address in the profile of the authenticated user.
-
+2. In the **Configure SAML response** section, configure your identity provider to send the SAML response. The SAML response must include the following attributes:
+   - `givenname`: BigAnimal uses the value as the given name in the profile of the authenticated user.
+   - `surname`: BigAnimal uses the value as the surname in the profile of the authenticated user.
+   - `name`: BigAnimal uses the value as the full name (`givenname` joined with `surname`) in the profile of the authenticated user.
+   - `emailaddress` (optional): BigAnimal uses the value for the email address in the profile of the authenticated user. 
+   
    Provide a `NameID` element in the `Subject` element. Provide your email ID as the value to `NameID`. BigAnimal uses the email ID you provide as your username and primary email, so format `NameID` like an email address.
 
    For example:
 
    ![](../images/nameID.png)
-3. On the **Setup Configuration** tab, enter the configuration information for your preferred SAML identity provider:
+3. In the **SAML settings** section, enter the configuration information for your preferred SAML identity provider:
 
    | Field      | Description |
    | ---------- | ----------- |
-   | Issuer URL | URL string uniquely associated with your identity provider partner. |
    | Single Sign-On URL | The identity provider's sign-on URL. |
-   | Identity provider signature certificate | Identity provider's assertion signing certificate (`.cer` or `.cert`). Coordinate with your identity provider partner to obtain this certificate securely. |
+   | Identity Provider Signature Certificate | Identity provider's assertion signing certificate (`.cer` or `.cert`). Coordinate with your identity provider partner to obtain this certificate securely. |
    | Request Binding | SAML Authentication Request Protocol binding used to send the authentication request: HTTP-Redirect,  HTTP-Post, or Hybrid (SAML request is REDIRECT and response is POST). |
    | Response Signature Algorithm (RSA) Algorithm | The signature algorithm used to sign the SAML AuthNRequest (RSA SHA-1 or RSA SHA-256). |
 4. Select **Test Connection**. If you connect to your identity provider successfully, your identity provider's login screen appears. If an error message appears, contact [Support](/biganimal/latest/overview/support).
 
 Once your identity provider is set up, you can view your connection status, ID, login URL, audience URI, and assertion consumer service URL from the BigAnimal portal on the Identity Provider page. Select **Admin > Identity Provider** to access it.
 
+## Add a domain
```

Comment:
Hi @ccestes we need the future link of this section, because we have a link on the UI that points to this section but it is current incorrect. 

@paige12345 

<img width="1139" alt="image" src="https://user-images.githubusercontent.com/79249661/232465218-bd2d1000-320b-42b3-bbf9-df81099e59ce.png">

Reply:
The link will be: /docs/biganimal/latest/getting_started/identity_provider/#add-a-domain